src/components/colorBlock/colorBlock.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { StyledColorBlock } from "./colorBlock.styles";
+import { Text } from "../styled";
+
+const ColorBlock = ({ text, string1, string2, string3, string4 }) => {
```

Comment:
instead of `string1, string2, string3, string4` can you pass in an array or strings called `colors` or something similar? so the variable has more meaning, and then for the style you can pass in `colors[0]` `colors[1]` etc

Reply:
also if you want to rename the `text` variable to `emotions` or something like that to have meaningful variable names